test/custom-write-binary/wasmscript.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Can we give this shell script a better name?

Also, we probably can't have shell scripts in the test setup since we need to be able to run the full suite under windows.